pkg/microservice/aslan/core/common/repository/models/wokflow_task_v4.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2021 The KodeRover Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package models
+
+import (
+	"go.mongodb.org/mongo-driver/bson/primitive"
+
+	"github.com/koderover/zadig/pkg/microservice/aslan/config"
+)
+
+type WorkflowTask struct {
+	ID           primitive.ObjectID `bson:"_id,omitempty"             json:"id,omitempty"`
+	TaskID       int64              `bson:"task_id"                   json:"task_id"`
+	WorkflowName string             `bson:"workflow_name"             json:"workflow_name"`
+	Status       config.Status      `bson:"status"                    json:"status,omitempty"`
+	Description  string             `bson:"description,omitempty"     json:"description,omitempty"`
```

Comment:
description should not be a task field. It should be a field of the workflow instead

Reply:
removed